<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSceneDlg</class>
 <widget class="QDialog" name="NewSceneDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NEW</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(96, 96, 96);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>GRID SIZE</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="optSize30">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
}

QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          0px;
}

QRadioButton::indicator:checked {
    background-color:       red;
        border:                 3px solid rgb(200,200,200);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(96,96,96);
    border:                 3px solid rgb(200,200,200);
}</string>
        </property>
        <property name="text">
         <string>30 X 30</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optSize60">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
}

QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          0px;
}

QRadioButton::indicator:checked {
    background-color:       red;
        border:                 3px solid rgb(200,200,200);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(96,96,96);
    border:                 3px solid rgb(200,200,200);
}</string>
        </property>
        <property name="text">
         <string>60 X 60</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optSize100">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
}

QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          0px;
}

QRadioButton::indicator:checked {
    background-color:       red;
        border:                 3px solid rgb(200,200,200);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(96,96,96);
    border:                 3px solid rgb(200,200,200);
}</string>
        </property>
        <property name="text">
         <string>100 X 100</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>SPACING</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="optSpacing8">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
}

QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          0px;
}

QRadioButton::indicator:checked {
    background-color:       red;
        border:                 3px solid rgb(200,200,200);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(96,96,96);
    border:                 3px solid rgb(200,200,200);
}</string>
        </property>
        <property name="text">
         <string>8 X 8</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optSpacing16">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
}

QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          0px;
}

QRadioButton::indicator:checked {
    background-color:       red;
        border:                 3px solid rgb(200,200,200);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(96,96,96);
    border:                 3px solid rgb(200,200,200);
}</string>
        </property>
        <property name="text">
         <string>16 X 16</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="optSpacing32">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton {
}

QRadioButton::indicator {
    width:                  12px;
    height:                 12px;
    border-radius:          0px;
}

QRadioButton::indicator:checked {
    background-color:       red;
        border:                 3px solid rgb(200,200,200);
}

QRadioButton::indicator:unchecked {
    background-color:       rgb(96,96,96);
    border:                 3px solid rgb(200,200,200);
}</string>
        </property>
        <property name="text">
         <string>32 X 32</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnNew">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(182, 182, 182);
border:none;
}
QPushButton:hover{
color: rgb(0, 0, 0);
background-color: rgb(200, 200, 200);
border:none;
}
QPushButton:pressed{
color: rgb(0, 0, 0);
background-color: rgb(210, 210, 210);
border:none;
}
QPushButton:disabled {
color: rgb(96, 96, 96);
background-color: rgb(200, 200, 200);
border:none;
}</string>
        </property>
        <property name="text">
         <string>NEW</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnNew</sender>
   <signal>released()</signal>
   <receiver>NewSceneDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
